### ISSUE TITLE
Extra Coin Icons

### ISSUE THREAD

**Issue by sumpfork** [sumpfork/dominiontabs#439]
[From BGG](https://boardgamegeek.com/thread/926575/article/41464985#41464985):
```
However, it appears some of the tabs (Abundance, Buried Treasure, Cage, Gondola, and Rope) have a hidden coin icon tucked in the bottom left. I don't think it's anything I've done, but if you have any solution before I print these, I'd appreciate it!
```
![image](https://user-images.githubusercontent.com/9730025/211123476-1cd35603-1127-4aed-8a81-b1a814e8a2d1.png)

May already be addressed by pending PR by @bszonye, I'll check on the weekend.

**Comment by bszonye** [comment]
Yes, #436 fixes this also.

**Comment by bszonye** [comment]
By the way, this bug wasn't caused by #413. The coin is embedded in the treasure-duration image: https://github.com/sumpfork/dominiontabs/blob/master/src/domdiv/images/treasure-duration.png

**Comment by sumpfork** [comment]
Fixed with #436
